galaxy_ng/app/api/ui/viewsets/user.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework import mixins
+from rest_framework.response import Response
```

Comment:
Flake8:

galaxy_ng/app/api/ui/viewsets/user.py:2:1: F401 'rest_framework.response.Response' imported but unused


It doesn't seem that flake8 is executed against PRs on Travis. Reported it as a bug #129